appveyor.yml

```diff
@@ -5,7 +5,7 @@ clone_folder: c:\gopath\src\github.com\spacemeshos\go-spacemesh\
 environment:
   GOPATH: c:\gopath
 
-stack: go 1.21
+stack: go 1.22
```

Comment:
I think this file can be deleted, I don't think we are using appveyor?

Reply:
I have no idea what this is :shrug:

Reply:
Removed.